README.md

```diff
@@ -1,3 +1,5 @@
+# THIS REPOSITORY HAS BEEN MOVED TO [http://github.com/fair-chem/fairchem/tree/main/src/fairchem/demo/ocpapi](http://github.com/fair-chem/fairchem/tree/main/src/fairchem/demo/ocpapi)
```

Comment:
A couple questions...

1. Should the trailing "/ocpapi" be removed from these links? http://github.com/fair-chem/fairchem/tree/main/src/fairchem/demo goes straight to the readme, which is probably useful.
2. Are there any followups to this diff? Or is the next step to archive the repo?